factories_ZA/basePlotter.py

```diff
@@ -551,7 +562,100 @@ def get_jet_flavour_cut(flav, jet_idx):
                         'binning': '(50, 0, 1)'
                 }
             ])
-            
+
+
+            #PLOTS IN ELLIPSE
+            if not self.btag or cat == "MuEl":
+                continue
+            if cat == "MuMu":
+                with open(self.filename_MuMu) as f:
+                    parameters = json.load(f)
+            elif cat == "ElEl":
+                with open(self.filename_ElEl) as f:
+                    parameters = json.load(f)
+            for j, line in enumerate(parameters):
+                if cat == "MuMu":
+                    inWindowCut = "window_MuMu.isInWindow({0}, {1}, 2., {2}, {3})".format(float(line[0]), float(line[1]), self.jj_str + ".M()", self.baseObject+".p4.M()")
```

Comment:
so this means `rho = 2.` right ? Could you maybe define this as an explicit parameter somewhere instead ? We may be asked to perform some tests during the review, you never know
(same comment for out of ellipse, this parameter should probably be common for the two cases)

Reply:
Yes, I agree